### PR TITLE
A lot of phrase adjustments again, added trailing dots, removed "allowed" from some descriptions.. etc

### PR DESCRIPTION
**Pull request by Smooey**

### CODE REVIEW

File: wcfsetup/install/lang/en.xml [hunk 1]
```diff
@@ -1204,8 +1204,8 @@ GmbH=Gesellschaft mit beschränkter Haftung]]></item>
 		<item name="wcf.acp.package.validation"><![CDATA[Validation Result]]></item>
 		<item name="wcf.acp.package.validation.errorCode.1"><![CDATA[The file “{$archive}”{if !$targetArchive|empty} within the archive “{$targetArchive}”{/if} is missing.]]></item>
 		<item name="wcf.acp.package.validation.errorCode.2"><![CDATA[The package “{$archive}” is missing the required “package.xml” file.]]></item>
-		<item name="wcf.acp.package.validation.errorCode.3"><![CDATA[The package name “{$packageName}” does not meet the expectations and was rejected.]]></item>
-		<item name="wcf.acp.package.validation.errorCode.4"><![CDATA[The package version “{$packageVersion}” does not meet the expectations and was rejected.]]></item>
+		<item name="wcf.acp.package.validation.errorCode.3"><![CDATA[The package name “{$packageName}” does not met the expectations and was rejected.]]></item>
+		<item name="wcf.acp.package.validation.errorCode.4"><![CDATA[The package version “{$packageVersion}” does not met the expectations and was rejected.]]></item>
```
Comment (Gravatronics) [inline wcfsetup/install/lang/en.xml]: That looks wrong to me.

**Comment by Smooey** [comment]
Which part? The typo "met", should be "meet"? Or...

I told Alex there was typos and so forth, but he didn't want those touched per se, and to be put back to original way (wrong ones ; typos, grammatically off, etc).

File: wcfsetup/install/lang/en.xml (hunk 1 above)
Comment (Smooey) [inline wcfsetup/install/lang/en.xml]: Which part? The typo "met", should be "meet"? Or...

I told Alex there was typos and so forth, but he didn't want those touched per se, and to be put back to original way.

Comment (Gravatronics) [inline wcfsetup/install/lang/en.xml]: Oh, I am sorry. I did not realize that the current state is `met` and that this change was the second commit in your pull request and that you changed `met -> meet` in the first commit, so never mind :smiley:

**Pull request merged**